pages/common/git-count.md

```diff
@@ -0,0 +1,13 @@
+# git count
+
+> Print the total number of all commits.
```

Comment:
`all` is implicit, no? 😇 
```suggestion
> Print the total number of commits.
```

Reply:
BTW, what are "all commits"? 🤔

Reply:
It was just an error. using the total beforehand makes the usage seem wrong, thanks for finding it